mlir/include/mlir/IR/Operation.h

```diff
@@ -320,7 +320,7 @@ class alignas(8) Operation final
 
   void print(raw_ostream &os, const OpPrintingFlags &flags = std::nullopt);
   void print(raw_ostream &os, AsmState &state);
-  void dump();
+  void dump(bool assumeVerified = false);
```

Comment:
If we want to ensure `dump` is available, we should mark it with LLVM_DUMP_METHOD. Helps prevent the symbol from being killed off.

Reply:
+1, please use LLVM_DUMP_METHOD